qdrant-landing/themes/qdrant-2024/assets/css/partials/_hybrid-cloud-cases.scss

```diff
@@ -0,0 +1,64 @@
+@use '../helpers/functions' as *;
+
+.hybrid-cloud-cases {
+  padding-bottom: $spacer * 4;
+  background-color: $neutral-94;
+
+  &__cards {
+    flex-direction: column;
+    gap: $spacer * 2;
+  }
+
+  &__card {
+    display: flex;
+    flex-direction: column;
+    gap: $spacer;
+
+    &-icon {
+      height: $spacer * 1.5;
+      width: $spacer * 1.5;
+    }
+
+    &-title {
+      margin-bottom: 0;
+      font-size: rem(20);
+      line-height: rem(30);
+      color: $neutral-20;
+    }
+
+    &-description {
+      margin-bottom: 0;
+      color: $neutral-30;
+    }
+  }
+
+  @include media-breakpoint-up(xl) {
+    padding-top: $spacer * 2.5;
+    padding-bottom: $spacer * 6.5;
+
+    &__wrapper {
+      max-width: rem(1134);
```

Comment:
Is this needed? `.container` already gets work done, isn't it?

Reply:
Yes, this is not necessary, I will delete it in all my PRs

Reply:
Thank you so much!